workspaces/announcements/plugins/announcements-backend/src/router.test.ts

```diff
@@ -62,6 +63,12 @@ describe('createRouter', () => {
   const mockNotificationService = {
     send: jest.fn().mockImplementation(async () => {}),
   };
+  const auditorSuccessMock = jest.fn().mockResolvedValue(undefined);
+  const auditorFailMock = jest.fn().mockResolvedValue(undefined);
+  const auditorCreateEventMock = jest.fn().mockImplementation(async () => ({
+    success: auditorSuccessMock,
+    fail: auditorFailMock,
+  }));
```

Comment:
Not the end of the world, but `mockServices` provides a mocked auditor service for you. If we leave this, I might create an issue to see if someone wants to contribute a refactor to leverage mockServices in all tests missing it. It's a good opportunity for first time contributor.